### DISCUSSION TITLE
make error

### DISCUSSION THREAD

**Discussion by LitterWindwind** [NVlabs/instant-ngp#589] [Q&A]
VkPhysicalDeviceBufferDeviceAddressFeaturesEXT is not denied
![image](https://user-images.githubusercontent.com/35981299/167293048-36dd365d-dfba-46c2-9974-2f8e5c909a51.png)

**Comment by LitterWindwind** [comment]
CUDA version is 11.4   I change the version to sovle this problem. But I think its not the best solution.

**Reply by killnice** [reply]
My CUDA version is 11.4, but I still have this error

**Reply by Tom94** [reply]
Can you try pulling the latest commit from `master`, as well as updating submodules, and see if the error is resolved then?

**Reply by killnice** [reply]
> Can you try pulling the latest commit from `master`, as well as updating submodules, and see if the error is resolved then?

thanks !  I have updated the dlss.cu file， now the problem have been solved !

**Reply by tomer-grin** [reply]
This still doesn't work for me :\

Update:
Managed to solve using installation of Vulkan SDK (although it is optional according to your README.md)
specifically for my case:
https://packages.lunarg.com/#

**Comment by parkgw95** [comment]
I have the same problem while using CUDA 11.0:

![image](https://user-images.githubusercontent.com/8743001/167375962-0b913c7c-a57b-470f-b1bf-ef908c314578.png)
